src/libopensc/card-openpgp.h

```diff
@@ -66,17 +66,18 @@ typedef enum _pgp_access {		/* access flags for the respective DO/file */
 	WRITE_MASK	= 0x1F00
 } pgp_access_t;
 
-typedef enum _pgp_ext_caps {	/* extended capabilities/features: bit flags */
-	EXT_CAP_ALG_ATTR_CHANGEABLE	= 0x0004,
-	EXT_CAP_PRIVATE_DO		= 0x0008,
-	EXT_CAP_C4_CHANGEABLE		= 0x0010,
-	EXT_CAP_KEY_IMPORT		= 0x0020,
-	EXT_CAP_GET_CHALLENGE		= 0x0040,
-	EXT_CAP_SM			= 0x0080,
-	EXT_CAP_LCS			= 0x0100,
-	EXT_CAP_CHAINING		= 0x1000,
-	EXT_CAP_APDU_EXT		= 0x2000,
-	EXT_CAP_MSE			= 0x4000
+typedef enum _pgp_ext_caps { /* extended capabilities/features: bit flags */
+	EXT_CAP_KDF_DO = 0x0001,
```

Comment:
not sure why you're chaning the alignment here...

Reply:
because clang-format-linter complains about the format.

Reply:
i see